### PR TITLE
Container utilities and deps

### PR DESCRIPTION
**Pull request by satmandu**
- This PR contains prereqs and packages for docker, as well as for bst.
- Also rebuilds libdb to add `compat185` headers.
- While these packages build fine, they have NOT been tested on actual hardware.
- They could also be further rubified. Please DO edit the package files in this PR to make that happen.
- Please DO test these binaries and make any suggestions  for changes which might make them work better.
- The i686 builds for this are obviously partial. If there isn't a i686 binary, the package just didn't build for i686. I will spend no more time on getting these packages to work for i686.

Builds properly:
- [x] x86_64
- [x] armv7l
- [ ] i686 (some do, some don't)

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=docker CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/containerd.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_x86_64/containerd-1.6.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '3560445ba0e85f47cede9836cdcbfd34879251f60abb9ddb71cf8ad8e0cf403a',
+     armv7l: '3560445ba0e85f47cede9836cdcbfd34879251f60abb9ddb71cf8ad8e0cf403a',
+     x86_64: '6924a4ebf2d3f50a2e9d7f18e3c23a1733d9ad1441a5115e0810a070bc5a16e6'
+  })
+
+  git_hashtag "v#{version}"
```
Comment (uberhacker) [inline packages/containerd.rb:25]: Move to above the binaries?

File: packages/containerd.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  def self.install
+    FileUtils.mkdir_p %W[
+      #{CREW_DEST_PREFIX}/etc/elogind
+      #{CREW_DEST_MAN_PREFIX}/man5
+      #{CREW_DEST_MAN_PREFIX}/man8
+    ]
+    system "make PREFIX=#{CREW_PREFIX} DESTDIR=#{CREW_DEST_DIR} install"
+    system "install -Dm644 containerd.service #{CREW_DEST_PREFIX}/etc/elogind/containerd.service"
+    system "install -Dm644 man/*.8 -t #{CREW_DEST_MAN_PREFIX}/man8"
+    system "install -Dm644 man/*.5 -t #{CREW_DEST_MAN_PREFIX}/man5"
```
Comment (uberhacker) [inline packages/containerd.rb:53]:
```
FileUtils.install 'containerd.service', "#{CREW_DEST_PREFIX}/etc/elogind/containerd.service", mode: 0o644
FileUtils.install Dir['man/*.5'], "#{CREW_DEST_MAN_PREFIX}/man5", mode: 0o644
FileUtils.install Dir['man/*.8'], "#{CREW_DEST_MAN_PREFIX}/man8", mode: 0o644
```

File: packages/containers_common.rb [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      Dir.chdir 'skopeo' do
+        system "install -vDm 644 default-policy.json #{CREW_DEST_PREFIX}/etc/containers/policy.json"
+        system "install -vDm 644 default.yaml -t #{CREW_DEST_PREFIX}/etc/containers/registries.d/"
+      end
+      Dir.chdir 'storage' do
+        system "install -vDm 644 storage.conf -t #{CREW_DEST_PREFIX}/etc/containers/"
+        system "install -vDm 644 storage.conf -t #{CREW_DEST_PREFIX}/share/containers/"
+        system "install -vDm 644 docs/*.1 -t #{CREW_DEST_MAN_PREFIX}/man1/"
+        system "install -vDm 644 docs/*.5 -t #{CREW_DEST_MAN_PREFIX}/man5/"
+      end
```
Comment (uberhacker) [inline packages/containers_common.rb:114]: Change all `system "install...` to `FileUtils.install ...`?

File: packages/docker.rb [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    system "install -Dm644 'src/github.com/docker/docker/contrib/init/systemd/docker.socket' #{CREW_DEST_PREFIX}/etc/elogind/docker.socket"
+    system "install -Dm644 'src/github.com/docker/docker/contrib/udev/80-docker.rules' #{CREW_DEST_PREFIX}/etc/udev/rules.d/80-docker.rules"
+    @docker_sysusers = 'g chronos - -'
+    File.write("#{CREW_DEST_PREFIX}/etc/sysusers.d/docker.conf", @docker_sysusers, perm: 0o644)
+    system "install -Dm755 build/docker #{CREW_DEST_PREFIX}/bin/docker"
+    system "install -Dm644 'contrib/completion/bash/docker' #{CREW_DEST_PREFIX}/share/bash-completion/completions/docker"
+    system "install -Dm644 'contrib/completion/zsh/_docker' #{CREW_DEST_PREFIX}/share/zsh/site-functions/_docker"
+    system "install -Dm644 'contrib/completion/fish/docker.fish' #{CREW_DEST_PREFIX}/share/fish/vendor_completions.d/docker.fish"
+    FileUtils.cp_r Dir.glob('man/man*'), "#{CREW_DEST_MAN_PREFIX}/"
+    system "install -Dm755 src/github.com/docker/buildx/docker-buildx #{CREW_DEST_LIB_PREFIX}/docker/cli-plugins/docker-buildx"
```
Comment (uberhacker) [inline packages/docker.rb:148]: Change all `system "install...` to `FileUtils.install ...`?

File: packages/netavark.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    @carch = ARCH == 'aarch64' || ARCH == 'armv7l' ? 'armv7-unknown-linux-gnueabihf' : "#{ARCH}-unknown-linux-gnu"
+    system "cargo fetch --locked --target #{@carch}"
+    system 'CARGO_TARGET_DIR=target cargo build --frozen --release --all-features'
+    system 'make -C docs'
+  end
+
+  def self.install
+    FileUtils.mkdir_p %W[{CREW_DEST_LIB_PREFIX}/podman #{CREW_DEST_PREFIX}/share/doc/netavark]
+    system "install -vDm 755 target/release/netavark -t #{CREW_DEST_LIB_PREFIX}/podman/"
+    system "make DESTDIR=#{CREW_DEST_DIR} PREFIX=#{CREW_PREFIX} install -C docs"
```
Comment (uberhacker) [inline packages/netavark.rb:41]: `system "install...` after `system "make ...`?

File: packages/docker.rb [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/20.10.12_armv7l/docker-20.10.12-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/20.10.12_x86_64/docker-20.10.12-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'e1aab9dc93822aa0b2ccf2c87f943801ca927dad0635141b4c014d73ad482805',
+     armv7l: 'e1aab9dc93822aa0b2ccf2c87f943801ca927dad0635141b4c014d73ad482805',
+     x86_64: 'f9e243148e09d240855ffef68f04c4716142612d30f0d541c186840c7dc1114b'
+  })
+
+  git_hashtag "v#{version}"
```
Comment (uberhacker) [inline packages/docker.rb:25]: Move to above the binaries?

File: packages/netavark.rb (hunk 5 above)
Comment (satmandu) [reply packages/netavark.rb:41]: The make is just for the docs, but sure...

**Comment by uberhacker** [comment]
This is an epic work and greatly appreciated.  I'm curious to see how well it works on arm.

**Comment by satmandu** [comment]
So I've made some headway with debugging docker by installing https://github.com/gdraheim/docker-systemctl-replacement/blob/master/files/docker/systemctl3.py to /usr/local/bin/systemctl as a systemctl replacement.

I modify the python3 path therein, and also need to create a symlink from /usr/local/etc/elogind to {XDG_CONFIG_HOME}/systemd/user or maybe we can just set the systems service file location to that by default?

That way we can in theory do `systemctl start --user contained, etc."

also containerd needs to be hacked to use a `/usr/local` based path instead of /var/lib/containerd. Suggestions? Something FHS compliant inside share? /usr/local/share/containerd?

**Comment by satmandu** [comment]
For the record, `XDG_CONFIG_HOME=/usr/local/.config` so `{XDG_CONFIG_HOME}/systemd/user` would be `/usr/local/.config/systemd/user` ...

File: packages/containerd.rb (hunk 1 above)
Comment (satmandu) [reply packages/containerd.rb:25]: One of the scripts I use keeps screwing this up. :/

**Comment by satmandu** [comment]
This may be good for merge for testing?

File: packages/containers_common.rb [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+  def self.build
+    @image_version = 'v5.20.0'
+    @podman_version = 'v4.0.2'
+    @shortnames_version = 'v2022.02.08'
+    @skopeo_version = 'v1.6.1'
+    @storage_version = 'v1.38.2'
+    Dir.chdir 'docs' do
+      system 'for _man_page in *.md
+      do
+        go-md2man -in $_man_page -out ${_man_page//.md}
```
Comment (supechicken666) [inline packages/containers_common.rb:38]: Did you mean `${_man_page/.md/}`?

File: packages/bridge_utils.rb [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+     armv7l: '41f566515eee8a83ce368c61e912cb11fcb023da3dca2573ab45ca99c85a3b58',
+       i686: '3a0367f328261023cd4c9b5d3e464822197b2fdce828e170ca654fb2e497cbe7',
+     x86_64: 'bf7b7e4461e09843909ce3585e4b11a3187f38b09471170f35fdc75e4dc72929'
+  })
+
+  def self.build
+    system 'autoreconf -fvi'
+    system "./configure \
+      #{CREW_OPTIONS} \
+      --sbindir=#{CREW_PREFIX}/bin \
```
Comment (Zopolis4) [inline packages/bridge_utils.rb:32]: Shouldn't this be `--sbindir=#{CREW_PREFIX}/sbin \`?

File: packages/iproute2.rb [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+       struct qdisc_util *get_qdisc_kind(const char *str);
+       struct filter_util *get_filter_kind(const char *str);
+    IPROUTE2_FHS_EOF
+    File.write('0001-make-iproute2-fhs-compliant.patch', @iproute2_fhs_patch)
+    system 'patch -Np1 -i 0001-make-iproute2-fhs-compliant.patch'
+    system "sed -i 's/-Werror//' Makefile"
+  end
+
+  def self.build
+    system './configure'
```
Comment (Zopolis4) [inline packages/iproute2.rb:141]: Not `./configure #{CREW_OPTIONS}`?

File: packages/containers_common.rb (hunk 7 above)
Comment (satmandu) [reply packages/containers_common.rb:38]: Copy from https://github.com/archlinux/svntogit-community/raw/packages/containers-common/trunk/PKGBUILD

File: packages/bridge_utils.rb (hunk 8 above)
Comment (satmandu) [reply packages/bridge_utils.rb:32]: No. Many packages imported from Arch Linux will have sbin binaries put in bin. I think it is fine...

File: packages/iproute2.rb (hunk 9 above)
Comment (satmandu) [reply packages/iproute2.rb:141]: Good catch.

File: packages/containers_common.rb [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+  depends_on 'go_md2man' => ':build'
+
+  def self.build
+    @image_version = 'v5.20.0'
+    @podman_version = 'v4.0.2'
+    @shortnames_version = 'v2022.02.08'
+    @skopeo_version = 'v1.6.1'
+    @storage_version = 'v1.38.2'
+    Dir.chdir 'docs' do
+      system 'for _man_page in *.md
```
Comment (supechicken666) [inline packages/containers_common.rb:36]:
```ruby
Dir.glob('docs/*.md').each do |man_page|
  system "go-md2man -in #{man_page} -out #{File.basename(man_page, '.md')}"
end
```

File: packages/iproute2.rb (hunk 9 above)
Comment (satmandu) [reply packages/iproute2.rb]: Fixed and rebuilt.

File: packages/containers_common.rb (hunk 7 above)
Comment (satmandu) [reply packages/containers_common.rb:38]: Thoughts on that?

**Comment by satmandu** [comment]
`sudo containerd` now appears to work.

Still don't have dockerd starting yet properly. Some sort of grpc error.

File: packages/containers_common.rb [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+        system 'go-md2man -in pkg/hooks/docs/oci-hooks.5.md -out oci-hooks.5'
+      end
+      system "git clone --depth 1 --branch #{@shortnames_version} https://github.com/containers/shortnames.git"
+      system "git clone --depth 1 --branch #{@skopeo_version} https://github.com/containers/skopeo.git"
+      system "git clone --depth 1 --branch #{@storage_version} https://github.com/containers/storage.git"
+      Dir.chdir 'storage' do
+        FileUtils.mkdir_p 'tests/tools/build/'
+        FileUtils.ln_s "#{CREW_PREFIX}/bin/go-md2man", 'tests/tools/build/'
+        system 'make -C docs'
+      end
```
Comment (uberhacker) [inline packages/containers_common.rb:67]: Should `image` and `podman` be separate packages as dependencies?

File: packages/containers_common.rb [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+        do
+          go-md2man -in $_man_page -out ${_man_page//.md}
+        done'
+      end
+      system "git clone --depth 1 --branch #{@podman_version} https://github.com/containers/podman.git"
+      Dir.chdir 'podman' do
+        system 'go-md2man -in pkg/hooks/docs/oci-hooks.5.md -out oci-hooks.5'
+      end
+      system "git clone --depth 1 --branch #{@shortnames_version} https://github.com/containers/shortnames.git"
+      system "git clone --depth 1 --branch #{@skopeo_version} https://github.com/containers/skopeo.git"
```
Comment (uberhacker) [inline packages/containers_common.rb:61]: What are we doing with `shortnames` and `skopeo`?  It looks like they're cloned and left alone.

Comment (uberhacker) [reply packages/containers_common.rb:61]: Nevermind, I see them in the install section.

File: packages/containers_common.rb (hunk 11 above)
Comment (uberhacker) [reply packages/containers_common.rb:67]: You know?  The more I think about it, the more I like it all in this package.

File: packages/containerd.rb [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+  binary_sha256({
+    aarch64: 'ba0e2edf47204f1dfb0dc009cfe9e92eaa51e269dd07333783d33e9fb6b7a73d',
+     armv7l: 'ba0e2edf47204f1dfb0dc009cfe9e92eaa51e269dd07333783d33e9fb6b7a73d',
+     x86_64: 'a16adc6e1e3bf5ab88439c5dc6c31a7d49e91e93f3b9e1e6e3f41644ec23ae44'
+  })
+
+  depends_on 'docker_systemctl_replacement'
+  depends_on 'fuse_overlayfs'
+  depends_on 'runc'
+  depends_on 'go' => ':build'
```
Comment (supechicken) [inline packages/containerd.rb:29]: `=> :build` (seems most of the build dependency flags in this pr need to be fixed :) )

Comment (satmandu) [reply packages/containerd.rb:29]: I think this may be due to the import script used...

File: packages/containerd.rb [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+  depends_on 'btrfsprogs' => ':build'
+  depends_on 'libseccomp' => ':build'
+  depends_on 'containers_common' => ':build'
+  depends_on 'go_md2man' => ':build'
```
Comment (uberhacker) [inline packages/containerd.rb:33]: Symbols should not be quoted.  `:build` not `':build'`.

File: packages/libbpf.rb [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libbpf/0.7.0_x86_64/libbpf-0.7.0-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '3428813d83b350dc34a7d3b924234c27c85b1fcd09d62ee8ed9810607d93163f',
+     armv7l: '3428813d83b350dc34a7d3b924234c27c85b1fcd09d62ee8ed9810607d93163f',
+     x86_64: 'e9a7dadae061accb7a6a303be994d82bc16cac80c9bd5fb893277ae7b3ca5f47'
+  })
+
+  depends_on 'elfutils'
+  depends_on 'rsync' => ':build'
```
Comment (uberhacker) [inline packages/libbpf.rb:27]: `:build` not `':build'`?

File: packages/netavark.rb [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+  })
+  binary_sha256({
+    aarch64: '5278c97de9d0e0d550d74e59bcda1250789a02f111dd2c5ab6ac6661d1a3d8dc',
+     armv7l: '5278c97de9d0e0d550d74e59bcda1250789a02f111dd2c5ab6ac6661d1a3d8dc',
+       i686: 'd6b6e5ae80e4a06f894850f871ffdf57460a605a7ccfc24c8ecab69e667b14b2',
+     x86_64: '78b18d0e1c2271b376b9ed5173185b39e109dd50140d5f8ba71177d4b8f46410'
+  })
+
+  depends_on 'libgit2' => ':build'
+  depends_on 'mandown' => ':build'
```
Comment (uberhacker) [inline packages/netavark.rb:29]: `:build` not `':build'`?

File: packages/runc.rb [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+  })
+  binary_sha256({
+    aarch64: '9b52e4e308ee6f28bfcad10dc3aed23543d6c323199a8ebdd5d9262b80f0438d',
+     armv7l: '9b52e4e308ee6f28bfcad10dc3aed23543d6c323199a8ebdd5d9262b80f0438d',
+     x86_64: '5652d483b9512919ebd0afb8095dc267368307e6ae592f8edcbc87caeab41289'
+  })
+
+  depends_on 'libseccomp'
+  depends_on 'go' => ':build'
+  depends_on 'go_md2man' => ':build'
```
Comment (uberhacker) [inline packages/runc.rb:28]: `:build` not `':build'`?

File: packages/docker.rb [hunk 18]
```diff
@@ -0,0 +1,161 @@
...
+  depends_on 'docker_systemctl_replacement'
+  depends_on 'eudev'
+  depends_on 'fuse_overlayfs'
+  depends_on 'iproute2'
+  depends_on 'lvm2'
+  depends_on 'sqlite'
+  depends_on 'btrfsprogs' => ':build'
+  depends_on 'elogind' => ':build'
+  depends_on 'go' => ':build'
+  depends_on 'go_md2man' => ':build'
```
Comment (uberhacker) [inline packages/docker.rb:37]: `:build` not `':build'`?

File: packages/go_md2man.rb [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/go_md2man/2.0.1_armv7l/go_md2man-2.0.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/go_md2man/2.0.1_x86_64/go_md2man-2.0.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'e12a504971810dee421ca4657a4af3c1be3ca82d4e319896e48d180c708e7679',
+     armv7l: 'e12a504971810dee421ca4657a4af3c1be3ca82d4e319896e48d180c708e7679',
+     x86_64: 'e1980595f9ff429ed47c7d8a2f57e5bf8a20e1d83277c0331d330d98b8558e97'
+  })
+
+  depends_on 'go'
```
Comment (uberhacker) [inline packages/go_md2man.rb:26]: `depends_on 'go' => :build`?

File: packages/bst.rb [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/bst/1.0.0-rc1_armv7l/bst-1.0.0-rc1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/bst/1.0.0-rc1_armv7l/bst-1.0.0-rc1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/bst/1.0.0-rc1_x86_64/bst-1.0.0-rc1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '867f159bc4b7a6951bbc2cac8358bf313a4cccd0f62c7f56cbf2f96b69764964',
+     armv7l: '867f159bc4b7a6951bbc2cac8358bf313a4cccd0f62c7f56cbf2f96b69764964',
+     x86_64: 'df1255b0bc150e8a323396cdba09acfb16137495b2d1aa07860577ae2c7df842'
```
Comment (uberhacker) [inline packages/bst.rb:20]: No i686 binary?

File: packages/containerd.rb [hunk 21]
```diff
@@ -0,0 +1,159 @@
...
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_x86_64/containerd-1.6.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'ba0e2edf47204f1dfb0dc009cfe9e92eaa51e269dd07333783d33e9fb6b7a73d',
+     armv7l: 'ba0e2edf47204f1dfb0dc009cfe9e92eaa51e269dd07333783d33e9fb6b7a73d',
+     x86_64: 'a16adc6e1e3bf5ab88439c5dc6c31a7d49e91e93f3b9e1e6e3f41644ec23ae44'
```
Comment (uberhacker) [inline packages/containerd.rb:23]: No i686 binary?

Comment (satmandu) [reply packages/containerd.rb:23]: I didn't even bother generating one. I'm spending no effort on these hosting apps trying to get them to work on i686...

File: packages/runc.rb [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+      make runc man"
+  end
+
+  def self.install
+    FileUtils.mkdir_p %W[#{CREW_DEST_PREFIX}/bin/ #{CREW_DEST_MAN_PREFIX}/man8
+                         #{CREW_DEST_PREFIX}/share/bash-completion/completions]
+    system "install -Dm755 runc #{CREW_DEST_PREFIX}/bin/runc"
+    system "install -Dm644 contrib/completions/bash/runc \
+      #{CREW_DEST_PREFIX}/share/bash-completion/completions/runc"
+    system "install -m644 man/man8/*.8 #{CREW_DEST_MAN_PREFIX}/man8"
```
Comment (uberhacker) [inline packages/runc.rb:46]: Change `system "install...` to `FileUtils.install...`?

File: packages/libbpf.rb [hunk 23]
```diff
@@ -0,0 +1,41 @@
...
+    system "sed -i 's,PREFIX ?= /usr,PREFIX ?= #{CREW_PREFIX},g' src/Makefile"
+  end
+
+  def self.build
+    system 'make -C src'
+  end
+
+  def self.install
+    system "make -C src DESTDIR=#{CREW_DEST_DIR} LIBSUBDIR=#{ARCH_LIB} install install_headers"
+    system "install -Dm 644 README.md -t #{CREW_DEST_PREFIX}/share/doc/libbpf"
```
Comment (uberhacker) [inline packages/libbpf.rb:39]: Change `system "install...` to `FileUtils.install...`?

**Review by uberhacker — CHANGES_REQUESTED** [review]
See comments.

File: packages/containerd.rb (hunk 21 above)
Comment (uberhacker) [reply packages/containerd.rb:23]: Yeah, I'm sure Docker won't work very well on 2 gb of memory max anyway.

**Comment by uberhacker** [comment]
Rubocop is whining about the extra escape characters.

**Comment by satmandu** [comment]
> Rubocop is whining about the extra escape characters.

Yeah I need to update some of these packages. They are out of date...

**Comment by satmandu** [comment]
superseded...

**Pull request closed without merging**